extern/storage-sealing/sector_state.go

```diff
@@ -2,6 +2,8 @@ package sealing
 
 type SectorState string
 
+var ExistSectorStateList = make(map[SectorState]struct{})
```

Comment:
Initializing the values here instead of in init would be cleaner / safer
```suggestion
var ExistSectorStateList = map[SectorState]struct{}{
	Empty: {},
...
}
```